spec/models/recipe_spec.rb

```diff
@@ -0,0 +1,22 @@
+require 'rails_helper'
+
+RSpec.describe Recipe, type: :model do
+  let(:recipe) {FactoryBot.create(:recipe)}
+
+  describe 'Database table' do
+    it {is_expected.to have_db_column :title}
+    it { should validate_presence_of(:title) }
+    it {is_expected.to have_db_column :description}
+    it { should validate_presence_of(:description) }
+    it {is_expected.to have_db_column :ingredients}
+    it { should validate_presence_of(:ingredients) }
+    it {is_expected.to have_db_column :directions}
+    it { should validate_presence_of(:directions) }
```

Comment:
Please do not use ‘should ‘ but rather ‘is_expected.to’. Also, group the validation specs in a separate ‘describe’ block

Reply:
Done

Reply:
Done